docs/replica_bootstrap.rst

```diff
@@ -65,12 +65,25 @@ for outdated backup files. Some people prefer other backup solutions, such as ``
 others, or simply roll their own scripts. In order to accommodate all those use-cases Patroni supports running custom
 scripts to clone a new replica. Those are configured in the ``postgresql`` configuration block:
 
+.. code:: YAML
+
+    postgresql:
+        create_replica_methods:
+            - <method name>
+        <method name>:
+            command: <command name>
+            args:
+                - '<arguments>'
+            keep_data: True
+            no_params: True
+            no_master: 1
+
 .. code:: YAML
 
     postgresql:
         create_replica_methods:
             - wal_e
-            - basebackup
+            - pgbackrest
```

Comment:
basebackup is the default name for pg_basebackup. Why did you replace it?

Reply:
Thank you for catching wrong section delete.